### ISSUE TITLE
RCS1214 should respect FormattableString

### ISSUE THREAD

**Issue by hankovich** [JosefPihrt/Roslynator#772]
**Product and Version Used**: Roslynator.Analyzers  Version="3.1.0"

**Steps to Reproduce**:
1. Create a method that accepts [FormattableString](https://docs.microsoft.com/en-us/dotnet/api/system.formattablestring?view=netstandard-2.0).
2. Call it and pass any constant string prefixed with `$`.

**Actual Behavior**:
```csharp
private static string Call(FormattableString s) => Call($"");
```
RCS1214 is reported and fix (Remove `$`) is suggested.

**Expected Behavior**:
After applying the proposed fix code is not compiling anymore.

```csharp
private static string Call(FormattableString s) => Call("");
```
